content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+
+### (Opsional) Mengendalikan klaster dari mesin selain simpul _control-plane_
+
+Untuk membuat kubectl bekerja pada mesin lain (e.g. laptop) agar dapat berbicara dengan
+klaster, kamu harus menyalin berkas kubeconfig administrator dari simpul _control-plane_
+ke _workstation_ seperti berikut:
+
+```bash
+scp root@<control-plane-host>:/etc/kubernetes/admin.conf .
+kubectl --kubeconfig ./admin.conf get nodes
+```
+
+{{< note >}}
+Contoh di atas mengasumsikan akses SSH dinyalakan untuk _root_. Jika tidak berlaku
+demikian, kamu dapat menyalin berkas `admin.conf` untuk dapat diakses oleh pengguna lain
+dan `scp` menggunakan pengguna lain tersebut.
+
+Berkas `admin.conf` memberikan penggunanya privilese _superuser_ terhadap klaster.
+Berkas ini harus digunakan seperlunya. Untuk pengguna biasa, direkomendasikan
+untuk membuat kredensial unik dengan privilese whitelist. Kamu dapat melakukan
+ini dengan perintah `kubeadm alpha kubeconfig user --client-name <CN>`
+. Perintah tersebut akan mencetak berkas KubeConfig ke STDOUT yang harus kamu simpan
+ke dalam sebuah berkas dan mendistribusikannya pada para pengguna. Setelah itu, whitelist
+privilese menggunakan `kubectl create (cluster)rolebinding`.
+{{< /note >}}
+
+### (Opsional) Meneruskan Peladen API ke localhost
+
+Jika kamu ingin terhubung dengan peladen API dari luar klaster kamu dapat menggunakan
+`kubectl proxy`:
+
+```bash
+scp root@<control-plane-host>:/etc/kubernetes/admin.conf .
+kubectl --kubeconfig ./admin.conf proxy
+```
+
+Kini kamu dapat mengakses peladen API secara lokal melalui `http://localhost:8001/api/v1`
+
+## Pembongkaran {#pembongkaran}
+
+Jika kamu menggunakan peladen sekali pakai untuk membuat klaster, sebagai ujicoba, kamu dapat
+mematikannya tanpa perlu melakukan pembongkaran. Kamu dapat menggunakan
+`kubectl config delete-cluster` untuk menghapus referensi lokal ke
+klaster.
+
+Namun, jika kamu ingin mengatur ulang klaster lebih rapih, pertama-tama kamu
+harus [menguras simpul](/docs/reference/generated/kubectl/kubectl-commands#drain)
+dan memastikan simpul sudah kosong, kemudian mengembalikan pengaturan pada simpul kembali seperti semula.
+
+### Menghapus simpul
+
+Berinteraksi dengan simpul _control-plane_ menggunakan kredensial yang sesuai, jalankan:
+
+```bash
+kubectl drain <node name> --delete-local-data --force --ignore-daemonsets
+kubectl delete node <node name>
+```
+
+Lalu, pada simpul yang dihapus, atur ulang semua kondisi `kubeadm` yang telah dipasang:
+
+```bash
+kubeadm reset
+```
+
+Proses pengaturan ulang tidak mengatur ulang atau membersihkan kebijakan iptables atau tabel IPVS. Jika kamu ingin mengatur ulang iptables, kamu harus melakukannya secara manual:
+
+```bash
+iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
+```
+
+Jika kamu ingin mengatur ulang tabel IPVS, kamu harus menjalankan perintah berikut:
+
+```bash
+ipvsadm -C
+```
+
+Jika kamu ingin memulai kembali, cukup jalankan `kubeadm init` atau `kubeadm join` dengan
```

Comment:
```suggestion
Jika kamu ingin mengulang dari awal, cukup jalankan `kubeadm init` atau `kubeadm join` dengan
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc